pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "spdx-tools"
+authors = [{name = "Ahmed H. Ismail", email = "ahm3d.hisham@gmail.com"}]
+maintainers = [
+    {name = "Philippe Ombredanne", email = "pombredanne@gmail.com"},
+    {name = "SPDX group at the Linux Foundation and others"},
+]
+license = {text = "Apache-2.0"}
+description = "SPDX parser and tools."
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+urls = {Homepage = "https://github.com/spdx/tools-python"}
+requires-python = ">=3.6"
+dependencies = ["ply", "rdflib", "click", "pyyaml", "xmltodict"]
+dynamic = ["version"]
+
+[project.scripts]
+pyspdxtools_convertor = "spdx.cli_tools.convertor:main"
+pyspdxtools_parser = "spdx.cli_tools.parser:main"
+
+[tool.setuptools]
+zip-safe = false
```

Comment:
`zip-safe` isnt important any more (c.f. https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html) , but it doesnt hurt to keep tracking this.
Could you add a comment here that this package isnt zip-safe because of https://github.com/spdx/tools-python/issues/257